pos_receipt_replace_user_by_trigram/models/res_users.py

```diff
@@ -14,4 +14,6 @@ class ResUsers(models.Model):
     @api.depends("firstname", "lastname")
     def _compute_pos_trigram(self):
         for user in self:
+            user.firstname = user.firstname or ""
```

Comment:
a bit confused by this assignation, is it necessary?
what about just adding `or ""` to functions params?

Reply:
yeah, you're right